src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -22,7 +22,11 @@ const MetaTrader = (() => {
         BinarySocket.wait('landing_company', 'get_account_status', 'statement').then(async () => {
             if (isEligible()) {
                 if (Client.get('is_virtual')) {
-                    await addAllAccounts();
+                    try {
+                        await addAllAccounts();
```

Comment:
how about the second usage of this on line 33?

Reply:
Done.

Reply:
what is done? 😅 I see no change committed